components/event-gateway/pkg/nats/authenticator.go

```diff
@@ -116,7 +128,7 @@ func (a *automateAuthenticator) Check(client natsd.ClientAuthentication) bool {
 	}
 
 	err := a.checkToken(token)
-	if err == nil {
+	if err == nil { // success
```

Comment:
I would change the logic in this `if` instead of adding a comment. 😅 (just my humble opinion)